README.md

```diff
@@ -31,63 +42,153 @@ envs:
+| `failOnTimeout`                    | _optional_  | A boolean flag that fails the CI job if at least one test exceeds the default test timeout. The default is set to `true`.                                                                                             |
+| `files`                            | _optional_  | Glob patterns to detect Synthetic test [configuration files][B-2].                                                                                                                                                    |
+| `jUnitReport`                      | _optional_  | The filename for a JUnit report if you want to generate one.                                                                                                                                                          |
+| `locations`                        | _optional_  | String of locations separated by semicolons to override the locations where your tests run.                                                                                                                           |
+| `mobileApplicationVersion`         | _optional_  | Override the default mobile application version for a Synthetic mobile application test. The version must be uploaded and available within Datadog.                                                                   |
+| `mobileApplicationVersionFilePath` | _optional_  | Override the application version for Synthetic mobile application tests.                                                                                                                                              |
+| `pollingTimeout`                   | _optional_  | The duration (in milliseconds) after which the batch is deemed as failed because of a time out. The default is 30 minutes.                                                                                            |
+| `publicIds`                        | _optional_  | String of public IDs separated by commas for Synthetic tests you want to trigger.                                                                                                                                     |
+| `site`                             | _optional_  | The Datadog site to send data to. If the `DD_SITE` environment variable is set, it takes precedence.                                                                                                                  |
+| `subdomain`                        | _optional_  | The name of the custom subdomain set to access your Datadog application. If the URL used to access Datadog is `myorg.datadoghq.com`, the `subdomain` value needs to be set to `myorg`.                                |
+| `testSearchQuery`                  | _optional_  | Trigger tests corresponding to a [search][C-5]  query. This can be useful if you are tagging your test configurations. See [best practices][C-7]  for more information on tagging.                                    |
```

Comment:
```suggestion
| `testSearchQuery`                  | _optional_  | Trigger tests corresponding to a [search][C-5] query. This can be useful if you are tagging your test configurations. See [best practices][C-7]  for more information on tagging.                                    |
```